packages/flutter/test/rendering/proxy_box_test.dart

```diff
@@ -963,6 +963,15 @@ void main() {
     expect(debugPaintClipOval(Clip.none), paintsExactlyCountTimes(#drawPath, 0));
     expect(debugPaintClipOval(Clip.none), paintsExactlyCountTimes(#drawParagraph, 0));
   });
+
+  test('RenderProxyBox behavior can be mixed in along with another base class', () {
+    final RenderFancyProxyBox fancyProxyBox = RenderFancyProxyBox(fancy: 6);
+    // Box has behavior from its base class:
+    expect(fancyProxyBox.fancyMethod(), 36);
+    // Box has behavior from RenderProxyBox:
+    expect(fancyProxyBox.computeDryLayout(const BoxConstraints(minHeight: 8)),
+        const Size(0, 8));
```

Comment:
formatting nit: either leave this all on one line or format as:

```suggestion
    expect(
      fancyProxyBox.computeDryLayout(const BoxConstraints(minHeight: 8)),
      const Size(0, 8),
    );
```